db/seeds.rb

```diff
@@ -85,6 +85,9 @@
 puts "Creating replies..."
 load Rails.root.join('db', 'seeds', 'reply.rb')
 
+puts "Queuing flat post generation (will not update until jobs are run)"
+FlatPost.regenerate_all
+
```

Comment:
...didn't we have this in here. did it get removed for some reason or is it in one of the subfiles?

Reply:
aha, yes, it was in `db/seeds/posts.rb`, but I see you moved it, and it makes more sense here.